proposals/lip-0053.md

```diff
@@ -118,13 +117,25 @@ All CCMs from terminated chains have no effect, they are neither applied nor for
 
 #### First Cross-chain Update from a Sidechain
```

Comment:
Isn't this section "First Cross-chain Update from a Sidechain" rather a new section than a subsection of "Creating and Posting Cross-chain Update Transactions".

Reply:
I am trying not to change these kind of choices that were made by Maxime and already reviewed. In a sense, we can still think of it as part of the section "Creating and Posting Cross-chain Update Transactions".

Reply:
Okay.